src/Cache/CacheOperationTypes/CacheOperationTypes.php

```diff
@@ -290,6 +314,10 @@ public function value() : string
         return $this->value;
     }
 
+    public function __toString()
```

Comment:
Is this PHP convention? i.e. Are PHP developers expected to `echo cacheGetResponseHit object`?
Sorry I misunderstand how `__toString()` works, but I didn't see any method like this in .NET SDK.

Reply:
These are the PHP convention equivalent to the `ToString()` methods in .NET. I added one to each of the non-incubating SDK response types in https://github.com/momentohq/client-sdk-dotnet/pull/252. Your comment does raise the issue that these are **not** implemented in our incubating response types . . . I'll add a ticket for that on the .NET repo.